cpp/CMakeLists.txt

```diff
@@ -77,6 +77,16 @@ rapids_find_package(CUDAToolkit REQUIRED
     INSTALL_EXPORT_SET  cugraph-exports
     )
 
+if (BUILD_CUGRAPH_MTMG_TESTS)
+    if(NOT TARGET ucx::ucp)
+        find_package(ucx REQUIRED)
```

Comment:
Does UCP have its own CMake buildsystem now?

Reply:
I have no idea.  We install ucx/ucp/ucs as part of the conda environment, this should just find it or fail.

Reply:
Thanks. I just installed the `ucxx` package from Conda and see that it provides `ucxx-config.cmake` - that's what's needed for this to work.